src/routes/AppRoutes.tsx

```diff
@@ -5,6 +5,7 @@ import SignUp from '../pages/SignUp/SignUp';
 import About from '../pages/About/About';
 import ContactUs from '@/pages/ContactUs/ContactUs';
 import FAQs from '@/pages/FAQs/FAQs';
+import Corporate from '@/pages/Corporate/Corporate';
```

Comment:
This has been added twice - please could you delete one instance?

Reply:
I think I deleted this!